game/scripts/vscripts/abilities/twin/boss_twin_twin_empathy.lua

```diff
@@ -0,0 +1,7 @@
+LinkLuaModifier("modifier_boss_twin_twin_empathy_buff", "abilities/twin/modifier_boss_twin_twin_empathy.lua", LUA_MODIFIER_MOTION_NONE)
```

Comment:
The same could be done if it was DataDriven, but I think it would make more sense to make the ability give the modifier rather than adding the modifier in `twin.lua`. Then make the ability passive if it isn't actually cast for anything.